cli/src/commands/operation/abandon.rs

```diff
@@ -82,25 +81,36 @@ pub fn cmd_op_abandon(
                 1 => parent_ops.into_iter().next().unwrap(),
                 _ => return Err(user_error("Cannot abandon a merge operation")),
             };
-            (parent_op, op)
+            (parent_op, vec![op])
         };
 
-    if abandon_head_op == current_head_op {
+    if let Some(op) = abandon_head_ops
+        .iter()
+        .find(|op| current_head_ops.contains(op))
+    {
         return Err(user_error_with_hint(
-            "Cannot abandon the current operation",
+            format!(
+                "Cannot abandon the current operation {}",
+                short_operation_hash(op.id())
+            ),
             "Run `jj undo` to revert the current operation, then use `jj op abandon`",
```

Comment:
I suppose this isn't entirely valid when there are multiple op heads, right? I think it would attempt to undo the merge operation, which I think fails. Should we just not show the hint if there are multiple op heads?

Reply:
> Should we just not show the hint if there are multiple op heads?

Make sense.